src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_map.scss

```diff
@@ -83,14 +83,27 @@ rather than in response to user action.
     @include border-radius(50%);
     @include box-shadow(0 0 1px 1px $color-text-inverted);
     background: $color-brand-two-normal;
+    border: solid 10px $color-brand-two-normal;
     color: $color-text-inverted;
     text-align: center;
 
     &.is-active, &:hover, &:focus {
         background: $color-brand-one-introvert;
+        border: solid 10px $color-brand-one-introvert;
     }
 
-    &.marker-cluster {
+@media print {
+    .marker-cluster {
+        border: solid 20px $color-brand-two-normal;
```

Comment:
the 20px value is dynamically generated by Leaflet.markercluster. We should not hard-code it here. As far as I can see there is no way to determine `border-width` from `width` in CSS (e.g. `border-width: 100%` is invalid). So this approach is doomed to fail.